app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center,
+                 clean_power,
+                 average_rating,
+                 count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        suma = sum([self.calculate_washing_price(car) for car in cars
+                    if car.clean_mark < self.clean_power])
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
+                self.wash_single_car(car)
+        return suma
+
+    def calculate_washing_price(self, car):
+        return round(
+            (car.comfort_class * (self.clean_power - car.clean_mark)) *
+            (self.average_rating / self.distance_from_city_center), 1)
+
```

Comment:
U have problems with flake8: W504 or W503. To solve this problem's you have to distribute your furmula for other's variables and then this error will disappear.